test/test_unstructured_2d.jl

```diff
@@ -62,9 +62,9 @@ isdir(outdir) && rm(outdir, recursive=true)
 
   @trixi_testset "elixir_ape_gauss_wall.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_ape_gauss_wall.jl"),
-      l2   = [0.029331247985489625, 0.02934616721732521, 0.03803253571320854, 0.0,
-              7.465932985352019e-16, 1.4931865970704038e-15, 1.4931865970704038e-15],
-      linf = [0.3626825396196784, 0.3684490307932018, 0.8477478712580901, 0.0,
+      l2   = [0.02933124798548954, 0.029346167217325133, 0.03803253571320853, 0.0,
```

Comment:
Why did this change? If I understand correctly, the old and the results should be identical, since in contrast to the Euler slip wall BCs, the wall BC "strategy" was not changed, only refactored. Or am I missing something?

Reply:
it seems to only be changing the last "round off" area of digits. This is probably due to me removing the computation of `u_tangent` and just directly projecting out the velocities. This would be identical in exact arithmetic, but not necessarily on a computer

Reply:
Ah ok, got it. Thanks